dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/commons_text/StringEscapeUtils.java

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
What changed here?

Reply:
This class added to our code base. However this exists in CXF 3.x too. Removing the class.